react-blocks/src/blocks/Covers/Covers.js

```diff
@@ -104,44 +241,50 @@ export class Covers extends Component {
             />
           </div>
 
-          <div>
-            <FormTokenField
-              value={ this.state.tagTokens }
-              suggestions={ tagSuggestions }
-              label='Select Tags'
-              onChange={ tokens => this.onSelectedTagsChange(tokens) }
-              placeholder="Select Tags"
-            />
-            <p class='FieldHelp'>Associate this block with Actions that have specific Tags</p>
-          </div>
+          {
+            this.props.posts !== 'undefined' && this.props.posts.length === 0
+              ?
+              <div>
+
+                <FormTokenField
+                  value={this.state.tagTokens}
+                  suggestions={tagSuggestions}
+                  label='Select Tags'
+                  onChange={tokens => this.onSelectedTagsChange(tokens)}
+                  placeholder="Select Tags"
+                />
+                <p class='FieldHelp'>Associate this block with Actions that have specific Tags</p>
+              </div>
+              : null
+          }
 
           {
-            this.props.cover_type === 3
-            ? <FormTokenField
-              value={ this.state.postTypeTokens }
-              suggestions={ postTypeSuggestions }
-              label='Post Types'
-              onChange={ tokens => this.onSelectedPostTypesChange(tokens) }
-              placeholder="Select Tags"
-            />
-            : null
+            this.props.cover_type === 3 && this.props.posts.length === 0
```

Comment:
Hey @kirdia looks like these conditions are false because `cover_type` is a String now, it `=== '3'`